### DISCUSSION TITLE
Setting up NextUI for Next.JS 13

### DISCUSSION THREAD

**Discussion by ezanglo** [heroui-inc/heroui#1099] [Help]
Hi! I'm currently learning react, nextjs and would want to use NextUI. 
I can see that the 2.0 of NextUI is supposed to be working for nextjs 13 but I wonder if I can use the current one and just upgrade to nextui 2.0 when it comes.

That said, all the other tutorials even the beta docs don't seem to use app directory of nextjs13 (experimental) 
I see from beta docs that you need to add 'use client' at the top of my file, but I don't know which file since I dont have any _app.tsx in this new app directory structure.

I'm sorry if I'm asking too much but can someone help me or give an example of how I can set this up?
Thanks in advance!

**Comment by ezanglo** [comment]
I did more reading and searching on the NextJS documentation and it said that pages/_app.tsx and pages/_directory.tsx are now replaced by app/layout.tsx, so I did use the NextUIProvider on that file and added 'use client'; on the app/page.tsx (from what i understand it should be put on the component), and used a NextUI component (Button). 

When I compiled it, it gave me 
`
error - SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
`

am i doing something wrong?

**Comment by spencerchang** [comment]
same question.
Because next.js 13 solve some important issues. I wanna using it.
